tests/test_main.py

```diff
@@ -138,9 +110,9 @@ def test_removed_comment():
 
 
 def test_double_quotes_instead_of_single():
-    with open(path_to_main(), "r") as file:
+    with open(path_to_main(), "r", encoding="utf-8") as file:
         main_content = file.read()
 
         assert (
             "'" not in main_content
-        ), "You have to use a double quotes \"\" instead of single ''"
+        ), "You have to use double quotes \"\" instead of single ''"
```

Comment:
The test `test_double_quotes_instead_of_single` enforces the use of double quotes in the code. While this is a style preference, ensure that it aligns with the project's coding standards. If single quotes are acceptable, this test might not be necessary.